### PR TITLE
remove node-sass as production dependency (fixes #4)

### LINKED ISSUE #4: take sass.null as argument instead of depending on node-sass directly
If npm or yarn installs two versions of node-sass (one as a dependency of this module, the other as a dependency of the consuming package), then there are cases where the node-sass.NULL returned by this importer !== sass.NULL that the node-sass of the consuming repo's webpack build returns. This can result in build errors and undefined behavior.

Instead, this importer should take as an argument the value that should be returned when it determines returning sass.null is appropriate.

### PR DESCRIPTION
**Pull request by lzilioli**
See #4

### CODE REVIEW

**Comment by lzilioli** [comment]
This should be a major release, as it contains a breaking API change.

File: index.js [hunk 1]
```diff
@@ -62,6 +61,10 @@ function getImportPath(importCache, pantry, ingredientName) {
  * looked up in the locations named by `config.pantrySearchPaths`. The first
  * matching pantry found is cached and the above process performed.
  *
+ * @param {*} NODE_SASS_NULL The object to return when node-sass should do its
+ *                           thing. You should pass require('node-sass').NULL
```
Comment (lawnsea) [inline index.js]: Please wrap to a two character indent from the start of the `@param` (see line 70 in your patch for an example)

File: test/index.js [hunk 2]
```diff
@@ -143,29 +144,53 @@ tap.test('absolute paths are not handled', function (t) {
...
+	var importer = rouxSassImporter.bind();
+	t.throws(
+		importer,
+		'Throws if no arguments passed.'
+	);
+});
+
+tap.test('returns whatever NODE_SASS_NULL it is passed', function (t) {
+	t.autoend();
+
```
Comment (lawnsea) [inline test/index.js]: Please fix indentation: should be two spaces

File: package.json [hunk 3]
```diff
@@ -45,8 +46,7 @@
   "dependencies": {
     "@retailmenot/roux": "^1.0.1",
     "bluebird": "^3.0.6",
-    "lodash": "^3.10.1",
-    "node-sass": "^3.4.2",
+    "lodash": "^4.16.4",
```
Comment (lawnsea) [inline package.json]: Why are these unrelated dependencies updated?

File: test/index.js (hunk 2 above)
Comment (lawnsea) [inline test/index.js]: Hang on, I'm seeing weird rendering here. Lemme check the raw diff.

Comment (lzilioli) [inline test/index.js]: nowhere in this file are there two indentations. Eslint is happy with it as-is.

**Comment by lzilioli** [comment]
@lawnsea I address your feedback in the two commits I just pushed.

File: test/index.js (hunk 2 above)
Comment (lawnsea) [inline test/index.js]: Screenshot with `ts=4`:

screenshot 2016-10-19 14 34 45

**Pull request merged**